docs/src/utils/isDev.ts

```diff
@@ -0,0 +1 @@
+export const isDev = process.env.NODE_ENV === 'development';
```

Comment:
Since we are testing it as a feature flag, should we create a new env var, like `ff-react-native`? 


Also, we can wrap up all the code in some comment like 

```js
/* Start React-Native */ {code} /* End React-Native */
```

Then, it's easier to tell which code is a part of the feature flag, in case we need to clean it up.

We may not have to, but just some A/B testing experience. Open to discuss 😁 

You might already found it, but just ref it here. Some env vars we are using https://github.com/aws-amplify/amplify-ui/blob/main/docs/.env.example